proxy/src/http.rs

```diff
@@ -156,6 +165,82 @@ pub fn with_subscriptions(
     warp::any().map(move || crate::notification::Subscriptions::clone(&subscriptions))
 }
 
+/// Bundled input data for project registration, shared
+/// between users and orgs.
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RegisterProjectInput {
+    /// User specified transaction fee.
+    transaction_fee: registry::Balance,
+    /// Optionally passed coco id to store for attestion.
+    maybe_coco_id: Option<String>,
```

Comment:
Say if we changed this to `RadUrn`, would it be turned from a string into the correct type using `Deserialize`? :thinking: I'm wondering if it would be better to have stronger types on input.

Reply:
We should also make the coco_id required, but that's another issue: https://github.com/radicle-dev/radicle-upstream/issues/397.

Reply:
@rudolfs can we move this discussion to that ticket then?